spec/models/wunderground_history_spec.rb

```diff
@@ -54,22 +105,25 @@
       expect(return_object).to be_a WundergroundHistory
     end
 
+    # don't make this one a Wunderground::Error, it's from us
     it "raises errors if data requested prematurely" do
-      wunderground = Wunderground.new(ENV["WUNDERGROUND_KEY"])
-      time = Time.zone.parse('January 22, 2016 12:00:00 -04:00')
+      time = Time.zone.parse('January 23, 2016 at 6pm')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.